### DISCUSSION TITLE
Can't verify i'm a current student to be able to aply for the github copilot

### DISCUSSION THREAD

**Discussion by Zer0plusOne** [community/community#143363] [GitHub Education] [answered]
### Hi everyone,

Hello to anyone who reads this, I'm actually having some serious problems proving to GitHub that I'm a current student in the second grade of cybersecurity at ENTI-UB even after providing all the info they requested. I sent a proof of the tuition, a screenshot of the campus where my profile is listed as a student, and even the payment plan for this 2024-2025 course.

I wanted to contact support through mail, but I don't find any email address to contact them, and the "creating a support ticket" from the "docs" subdomain of GitHub just tells me that if I'm not paying for anything related to their products currently, I can't access that feature.

I'm just trying to learn and have a bit of help when I do some practice sessions or work on some projects. 

Sorry if i commited any mistakes writhing but im doing by best to type this without a traductor(I'm Spanish)

**Comment by ravindudil5han** [comment, marked as answer]
It sounds like a frustrating situation, and you're doing everything right by providing the documentation GitHub requested. Here’s a clear solution to help you get verified as a student on GitHub.

### 1. **Double-Check the Documentation Requirements**
Sometimes the verification process is picky about what they accept. GitHub typically requires:
   - An official document from your school, like an enrollment letter or an official transcript.
   - Your full name and a date showing you are enrolled for the current academic year.
   
Since you've already provided a tuition proof, campus screenshot, and payment plan, it may be useful to add an official enrollment confirmation or class schedule if available.

### 2. **Submit a GitHub Support Ticket**
You mentioned difficulty with submitting a support ticket since you're not a paying user, but there is a way to reach them:
   - Visit [GitHub Education Support](https://education.github.com/discount_requests/new).
   - This page is specifically for education-related requests and will let you submit a ticket even if you’re not a paying user.

   - **Tip:** Clearly explain your situation, including that you are in the second grade of cybersecurity at ENTI-UB, and list the documents you have already provided. 

### 3. **Alternative Contact Options**
If the ticket system doesn’t work, try these:
   - **Twitter Support:** GitHub has a support account on Twitter, [@GitHubHelp](https://twitter.com/githubhelp). They often respond quickly to queries here.
   - **GitHub Community Forum:** Post your issue in the [GitHub Community Forum](https://github.community/) under the **Education** section, explaining the trouble you're having with verification. Sometimes, GitHub staff or other community members will respond with specific advice.

### 4. **Wait and Reapply if Necessary**
If you’ve already submitted a request but haven’t heard back, give them a few days to review it, as the process can sometimes be slow. If they deny the application, reapply with any additional documentation you might be able to gather (such as your student ID or an official academic document).

### **Summary**
- Visit the GitHub Education Support page and submit a request there.
- Contact @GitHubHelp on Twitter if you don't get a response.
- Post in the GitHub Community Forum as another way to reach their team.

By following these steps, you should have a better chance of getting your account verified. Good luck, and feel free to reach out if you have more questions!
